### PR TITLE
Pypi devops pr version templates

### LINKED ISSUE #36: PR Version number Template for PyPi version publishing
Support is needed for templates for our Python packages.

### PR DESCRIPTION
**Pull request by SamVanhoutte**
Closes #36

### CODE REVIEW

**Comment by stijnmoreels** [comment]
Not sure what's the different with: https://github.com/arcus-azure/azure-devops-templates/blob/master/nuget/determine-pr-version.yml? 🤔

**Comment by tomkerkhove** [comment]
The version format that is being composed is different, so we'll be pragmatic and use a new template and will open an issue to refactor them and see what we can re-use.

**Comment by SamVanhoutte** [comment]
@stijnmoreels - pypi does not support dots/hyphens/underscores for their "dev releases", it seems
so the version number we're determining here is in the following format:
1.0.0.dev202004200025
- 1.0.0 > the pipeline variable
- dev > alpha/ci package
- buildnumber > date + 0000 format of PR number

File: README.md [hunk 1]
```diff
@@ -19,3 +19,8 @@ Collection of Azure DevOps YAML Templates used to build & ship Arcus.
 - [Determine Pull Request Version](nuget/determine-pr-version.yml): determines the package version based on the build number into a YAML variable called `$(packageVersion)`.
 - [Publish Preview NuGet Package](nuget/publish-preview-package.yml): allows you to push a preview NuGet package to our internal feed.
 - [Publish Official NuGet Package](nuget/publish-official-package.yml): allows you to push a new official NuGet package to our external feed.
+
+
```
Comment (tomkerkhove) [inline README.md]:
```suggestion
```

**Review by tomkerkhove — DISMISSED** [review]

**Comment by tomkerkhove** [comment]
> The version format that is being composed is different, so we'll be pragmatic and use a new template and will open an issue to refactor them and see what we can re-use.

Tracking in https://github.com/arcus-azure/azure-devops-templates/issues/38

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**